tests/certification/bindings/azure/storagequeues/storagequeue_test.go

```diff
@@ -312,7 +311,7 @@ func TestAzureStorageQueueForDecode(t *testing.T) {
 		// Declare the expected data.
 		msgs := make([]string, numOfMessages)
 		for i := 0; i < numOfMessages; i++ {
-			msgs[i] = fmt.Sprintf("Message %03d", i)
+			msgs[i] = fmt.Sprintf("Message 新 %03d", i) // the chinese character is part of the test for UTF-8 characters
```

Comment:
I normally use emojis 🤷

Reply:
I used Chinese because the impacted user had issues with Chinese characters. The character I used means "message" :)